errno/errname.go

```diff
@@ -1014,6 +1014,7 @@ var MySQLErrName = map[uint16]*mysql.ErrMessage{
 	ErrInconsistentHandle:            mysql.Message("writing inconsistent data in table: %s, index: %s, index-handle:%#v != record-handle:%#v, index: %#v, record: %#v", []int{2, 3, 4, 5}),
 	ErrInconsistentIndexedValue:      mysql.Message("writing inconsistent data in table: %s, index: %s, col: %s, indexed-value:{%s} != record-value:{%s}", []int{3, 4}),
 	ErrAssertionFailed:               mysql.Message("assertion failed: key: %s, assertion: %s, start_ts: %v, existing start ts: %v, existing commit ts: %v", []int{0}),
+	ErrInstanceScope:                 mysql.Message("allowing variable %s with INSTANCE scope change to SESSION scope while setting for now, but its deprecated", nil),
```

Comment:
I think this might be clearer / shorter:
```suggestion
	ErrInstanceScope:                 mysql.Message("modifying %s will require SET GLOBAL in a future version of TiDB", nil),
```